AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -208,25 +154,39 @@ - (void)initialDataLoadingWithAnimationOption:(ASDataControllerAnimationOptions)
 
 #pragma mark - Data Update
 
-- (void)beginUpdates {
+- (void)beginUpdates
+{
   dispatch_async([[self class] sizingQueue], ^{
     [self asyncUpdateDataWithBlock:^{
       _batchUpdateCounter++;
-      [_delegate dataControllerBeginUpdates:self];
     }];
   });
 }
 
 - (void)endUpdates {
+  [self endUpdatesWithCompletion:NULL];
+}
+
+- (void)endUpdatesWithCompletion:(void (^)(BOOL))completion
+{
   dispatch_async([[self class] sizingQueue], ^{
-    [self asyncUpdateDataWithBlock:^{
+    dispatch_async(dispatch_get_main_queue(), ^{
```

Comment:
for example, here, we're trampolining to the sizing queue and then immediately trampolining to the main queue — is this necessary?